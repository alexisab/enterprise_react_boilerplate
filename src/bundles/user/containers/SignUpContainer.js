import React from 'react';
// import { LoginFormComposer as LoginForm } from '../composers';
import SignUpForm from '../components/SignUpForm';
import '../../../css/sessions.css';

const SignUpContainer = ({ history }) => (
  <div className="sign-up">
    <h1 className="title">Sign Up</h1>
    <SignUpForm history={history} />
  </div>
);

export default SignUpContainer;
