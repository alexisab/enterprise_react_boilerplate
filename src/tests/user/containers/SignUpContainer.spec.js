import React from "react";
import renderer from "react-test-renderer";
import SignUpContainer from "../../../bundles/user/containers/SignUpContainer";

const history = {
  go: () => {},
  push: () => {}
};

describe("SignUpContainer", () => {
  test("it should match snapshot", () => {
    const component = renderer.create(<SignUpContainer history={history} />);

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
