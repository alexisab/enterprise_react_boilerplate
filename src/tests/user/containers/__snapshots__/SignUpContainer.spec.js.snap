// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUpContainer it should match snapshot 1`] = `
<div
  className="sign-up"
>
  <h1
    className="title"
  >
    Sign Up
  </h1>
  <div
    className="signup-form"
  >
    <form
      className="ant-form ant-form-horizontal"
      onSubmit={[Function]}
    >
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <span
                className="ant-input-affix-wrapper"
                style={undefined}
              >
                <span
                  className="ant-input-prefix"
                >
                  <i
                    className="anticon anticon-user"
                    style={
                      Object {
                        "fontSize": 13,
                      }
                    }
                  />
                </span>
                <input
                  className="ant-input"
                  disabled={false}
                  name="Name"
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Name"
                  style={null}
                  type="text"
                />
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <span
                className="ant-input-affix-wrapper"
                style={undefined}
              >
                <span
                  className="ant-input-prefix"
                >
                  <i
                    className="anticon anticon-mail"
                    style={
                      Object {
                        "fontSize": 13,
                      }
                    }
                  />
                </span>
                <input
                  className="ant-input"
                  disabled={false}
                  name="Email"
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Email"
                  style={null}
                  type="text"
                />
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <span
                className="ant-input-affix-wrapper"
                style={undefined}
              >
                <span
                  className="ant-input-prefix"
                >
                  <i
                    className="anticon anticon-lock"
                    style={
                      Object {
                        "fontSize": 13,
                      }
                    }
                  />
                </span>
                <input
                  className="ant-input"
                  disabled={false}
                  name="Password"
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Password"
                  style={null}
                  type="password"
                />
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <span
                className="ant-input-affix-wrapper"
                style={undefined}
              >
                <span
                  className="ant-input-prefix"
                >
                  <i
                    className="anticon anticon-lock"
                    style={
                      Object {
                        "fontSize": 13,
                      }
                    }
                  />
                </span>
                <input
                  className="ant-input"
                  disabled={false}
                  name="Password Confirmation"
                  onChange={[Function]}
                  onKeyDown={[Function]}
                  placeholder="Password Confirmation"
                  style={null}
                  type="password"
                />
              </span>
            </span>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item"
        style={undefined}
      >
        <div
          className="ant-form-item-control-wrapper"
        >
          <div
            className="ant-form-item-control"
          >
            <span
              className="ant-form-item-children"
            >
              <button
                className="ant-btn login-form-button ant-btn-primary"
                onClick={[Function]}
                type="submit"
              >
                <span>
                  Sign Up
                </span>
              </button>
              <div
                className="form-footer"
              >
                <a
                  href="/"
                >
                  Have an account? Login here
                </a>
              </div>
            </span>
          </div>
        </div>
      </div>
    </form>
  </div>
</div>
`;
